src/app/index.ts

```diff
@@ -56,13 +57,21 @@ const app = express();
 
 app.use(compression()); // Compress all routes
 // helps mitigate cross-site scripting attacks and other known vulnerabilities
+const reportOnly =
+  EXPRESS_CONTENT_SECURITY_POLICY_CONFIG['report-only'] || EXPRESS_CONTENT_SECURITY_POLICY_CONFIG.reportOnly;
+if (reportOnly !== undefined) {
+  delete EXPRESS_CONTENT_SECURITY_POLICY_CONFIG['report-only'];
```

Comment:
so that it works with both `report-only` and `reportOnly`, seems like to some extent helmetjs supports both syntaxes, see examples in [reference section](https://helmetjs.github.io/#reference).

The reportOnly is consumed differently from the other directives that is why am removing it.